src/utils/re_frame.cljs

```diff
@@ -78,3 +78,5 @@
 (def sub (comp deref re-frame/subscribe))
 
 (def dispatch re-frame/dispatch)
+
+(def reg-event-fx re-frame/reg-event-fx)
```

Comment:
@ilmotta added this so we still work through `rf/...` wdyt?

Reply:
Seems reasonable to me @J-Son89

Reply:
I'll make a follow up pr to adjust the other uses so 👍